build.sbt

```diff
@@ -75,17 +75,19 @@ lazy val core = projectMatrix
   .configure(catsEffectDependencies)
   .settings(
     libraryDependencies ++= Seq(
-      "com.eed3si9n.expecty" %%% "expecty"                % "0.14.1",
-      "org.portable-scala"   %%% "portable-scala-reflect" % "1.0.0"
+      "com.eed3si9n.expecty" %%% "expecty"                % "0.15.0",
+      ("org.portable-scala"  %%% "portable-scala-reflect" % "1.0.0").withDottyCompat(
+        scalaVersion.value)
     ),
     libraryDependencies ++= {
       if (virtualAxes.value.contains(VirtualAxis.jvm))
         Seq(
-          "org.scala-js" %%% "scalajs-stubs" % "1.0.0" % "provided"
+          ("org.scala-js" %%% "scalajs-stubs" % "1.0.0" % "provided").withDottyCompat(
```

Comment:
this dep is like 2 annotations, may be we can copypaste it?

Reply:
mmm not sure how we could do it without polluting the user's classpath tbh. I think having the dotty compat on this one is low risk for the time being, since it's a compile-only thing.